.github/workflows/workflow.yml

```diff
@@ -0,0 +1,55 @@
+name: envattrs workflow
+
+on: push
+
+permissions:
+  actions: write
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  pytest:
+    name: Run pytest
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.6
+      - run: |
+          curl -sSL https://install.python-poetry.org | python - \
+        env:
+          POETRY_VERSION: 1.1.15
+        shell: bash
+      - run: poetry install
+        shell: bash
```

Comment:
Btw., there's a poetry action out there.

Reply:
We used to use https://github.com/abatilo/actions-poetry, but it isn't up to date.
There's https://github.com/snok/install-poetry which is more complete.
IMO poetry install is simple enough not to warrant using external actions, but feel free to use it

Reply:
Yeah the thing is installing poetry is really a one liner, so adding external gha dependency seems like an overkill. And these actions still don't run the `poetry install` command either, which at least is a bit more useful.